src/reports/models/reportEmailSender.js

```diff
@@ -7,7 +7,7 @@ const fs = require('fs'),
     configHandler = require('../../configManager/models/configHandler'),
     configConsts = require('../../common/consts').CONFIG,
     logger = require('../../common/logger');
-module.exports.sendAggregateReport = async (aggregatedResults, job, emails) => {
+module.exports.sendAggregateReport = async (aggregatedResults, job, emails, reportBenchmark = {}) => {
```

Comment:
this is from the server:
(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'score' of undefined                                                                                                          │
│     at eval (lodash.templateSources[1]:44:44)                                                                                                                                                            │
│     at generateReportFromTemplate (/usr/src/reports/models/reportEmailSender.js:113:16)                                                                                                                  │
│     at Object.module.exports.sendAggregateReport (/usr/src/reports/models/reportEmailSender.js:32:20)                                                                                                    │
│     at handleDone (/usr/src/reports/models/notifier.js:117:27)                                                                                                                                           │
│     at <anonymous>                                                                                                                                                                                       │
│     at process._tickCallback (internal/process/next_tick.js:189:7)

Reply:
Was not merged to master, fixed